src/node/consumer/esbuild.ts

```diff
@@ -1,7 +1,15 @@
 import { build } from "esbuild";
 import { readFileSync, rmSync } from "fs";
 
-const { devDependencies } = JSON.parse(readFileSync("./package.json", "utf8"));
+interface DevDependencies {
+    [key: string]: string;
+}
```

Comment:
Could be a record too.

```suggestion
type DevDependencies = Record<string, string>
```